tests/test_other.py

```diff
@@ -9407,7 +9407,7 @@ def test(args):
     # Changing this option to [] should decrease code size.
     self.assertLess(changed, normal)
     # Check an absolute code size as well, with some slack.
-    self.assertLess(abs(changed - 5001), 150)
+    self.assertLess(abs(changed - 5174), 150)
```

Comment:
The rebase here is actually going from 5150 to 5174 so 24 bytes difference from `main`